roles/confluent.variables/vars/main.yml

```diff
@@ -632,13 +639,11 @@ kafka_rest_properties:
   defaults:
     enabled: true
     properties:
-      producer.interceptor.classes: io.confluent.monitoring.clients.interceptor.MonitoringProducerInterceptor
-      consumer.interceptor.classes: io.confluent.monitoring.clients.interceptor.MonitoringConsumerInterceptor
       listeners: "{{kafka_rest_http_protocol}}://0.0.0.0:{{kafka_rest_port}}"
       host.name: "{{inventory_hostname}}"
+      # TODO this prop is buggy
```

Comment:
this property has a loop with `zookeeper_final_properties.clientPort`
property is getting removed in 600 release, could resolve it there